graph/ThingGraph.java

```diff
@@ -309,7 +309,7 @@ public ThingVertex convertToReadable(VertexIID.Thing iid) {
                     return v;
                 }
         );
-        if (!isInferred && vertex.isInferred()) vertex.isInferred(false);
+        assert isInferred == vertex.isInferred();
```

Comment:
since we no longer support changing the vertex's inferred flag, we should assert that the state that is expected matches the vertex's actual status